composer.lock

```diff
@@ -1047,16 +1169,16 @@
         },
         {
             "name": "symfony/config",
-            "version": "v3.4.18",
+            "version": "v3.4.29",
```

Comment:
Setinkové updaty naštěstí nevadí, ale stejně by neměly být takhle "přibalené". Ale už to nech.